### ISSUE TITLE
Axonal spheroids

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7395]
**Preferred term label:**
Axonal spheroids

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://pubmed.ncbi.nlm.nih.gov/31775912/ 2/2

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
